pkg/operation/botanist/monitoring.go

```diff
@@ -97,6 +97,7 @@ func (b *Botanist) DeploySeedMonitoring(ctx context.Context) error {
 		b.Shoot.Components.SystemComponents.CoreDNS,
 		b.Shoot.Components.SystemComponents.KubeProxy,
 		b.Shoot.Components.SystemComponents.VPNShoot,
+		b.Shoot.Components.ControlPlane.ResourceManager,
```

Comment:
Nit: Move this up to the other lines tarting with `b.Shoot.Components.ControlPlane`